.github/workflows/build.yml

```diff
@@ -28,4 +28,24 @@ jobs:
         run: dotnet build --no-incremental /p:SignAssembly=true /p:AssemblyOriginatorKeyFile=../key.snk
 
       - name: Test SDK
-        run: dotnet test
+        run: dotnet test --configuration Release --verbosity normal --collect:"XPlat Code Coverage" --results-directory ./coverage -p:ExcludeByFile="**/GitHub/**/*.cs" 
+
+      - name: Code Coverage Report
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        with:
+          filename: coverage/**/coverage.cobertura.xml
+          badge: true
+          fail_below_min: false
+          format: markdown
+          hide_branch_rate: false
+          hide_complexity: true
+          indicators: true
+          output: both
+          thresholds: '60 80'
+  
+      - name: Add Coverage PR Comment
+        uses: marocchino/sticky-pull-request-comment@v2
```

Comment:
I'm also curious about this choice, since many other options exist here, like https://github.com/thollander/actions-comment-pull-request.

Reply:
As with the above - this was the favored implementation and streamlined option.  I'm not sure I am 100% sold on having a PR comment with coverage details - this is really try on the shoe and see if it fits. I am most likely going to pull this out and leave it as reporting in the CI step and leave bubbling up the results for another PR.